azure-pipelines.yml

```diff
@@ -22,13 +22,15 @@ jobs:
         py_3.7_32:
           PYTHON_VERSION: "3.7"
           PYTHON_ARCH: "x86"
-          NP_BUILD_DEP: "1.16.5"
-          NIGHTLY_BUILD: "true"
+          NP_BUILD_DEP: "1.17.3"
```

Comment:
we still need to build 1.3.1, 1.3.2, .... etc 

we won't be increasing min versions in patch releases, so this needs to reverted.

Reply:
1.17.3 is the minimum version for 1.3.x though, but it wasn't bumped when the PR that released 1.3 was merged. This PR is just syncing the min versions with what is present on the main repo. See https://github.com/pandas-dev/pandas/blob/1.3.x/pyproject.toml for example which is supported by the docs link in the OP.

Reply:
> 1.17.3 is the minimum version for 1.3.x though

thanks. is this PR ready to merge. The failures are unrelated?

Reply:
Yep. Thanks for the quick reviews :).